### ISSUE TITLE
Scrollspy issue 4.0.0

### ISSUE THREAD

**Issue by rasmuswoelk** [twbs/bootstrap#17676]
I'm having difficulties getting the scrollspy working in bootstrap 4.0.0.

Here is an example: http://pastebin.com/29f2mWuX

I am using the bootstrap libraries from bootstraps own documentation directly. Also their own jquery version. In the bottom you can see the out-commented version of bootstrap3. If you activate it the scrollspy works.

I pretty sure I have done the (simple) installation correctly but I might be wrong. Anyone else having this issue?

**Comment by twbs-lmvtfy** [comment]
Hi @rasmuswoelk!

You appear to have posted a live example (http://pastebin.com/DZSJHKMJ), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 22, column 2: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 32, column 2: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 87, column 3: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 142, column 2: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 197, column 3: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Superseded by #17679.
